src/olympia/blocklist/models.py

```diff
@@ -157,25 +157,24 @@ def get_blocks_from_guids(cls, guids):
         return blocks
 
 
+class BlockType(models.IntegerChoices):
+    BLOCKED = 0, _('Blocked')
```

Comment:
I know @diox had mentioned we might want to keep the emojified versions of the strings. Maybe @wagnerand also has an opinion. Is it possible we can use only one set of these, or it really must be two totally separate strings?

Reply:
Yes, we explicitly didn't use "Restricted" in admin tools because we already use that term internally for other admin-related features (see all the `*Restriction` models in `users/models.py`). The emojis were also added to visually differentiate between soft and hard blocks and you'll need @wagnerand to weigh in if you want to change this (with screenshots of the admin tools for various scenarios after that change)